### PR TITLE
Add cypress test for editing

### PR DESCRIPTION
**Pull request by miram13**
**Which issue(s) this PR fixes**
https://github.com/KaotoIO/kaoto-ui/issues/298

### CODE REVIEW

**Review by Delawen — APPROVED** [review]

**Comment by Delawen** [comment]
The failure with the job "Generate Docker image" is fixed on the main branch. It expects a username and password that are not on the developer's fork.

Unless we rebase this PR, it will continue failing. Which is not a problem, this PR is not affecting the docker image generation. We can just ignore it for this PR.

**Comment by miram13** [comment]
> The failure with the job "Generate Docker image" is fixed on the main branch. It expects a username and password that are not on the developer's fork.
> 
> Unless we rebase this PR, it will continue failing. Which is not a problem, this PR is not affecting the docker image generation. We can just ignore it for this PR.

**Comment by miram13** [comment]
Okay

**Comment by Delawen** [comment]
This PR seems to work now after rebasing :)

File: cypress/integration/5-editing/editing.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+describe('editing properties', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:1337');
```
Comment (Delawen) [inline cypress/integration/5-editing/editing.spec.js:3]: Can you change this to use the configured url too? @miram13

Comment (miram13) [reply cypress/integration/5-editing/editing.spec.js]: Ya sure

**Pull request closed without merging**